### ISSUE TITLE
h3 has low contrast ratio

### ISSUE THREAD

**Issue by Aliecake** [LeeVaughn/treehouse-hacktoberfesters#8]
###Accesibility issue

h3 & background have insufficient contrast ratios.

**Comment by LeeVaughn** [comment]
I'll look at maybe changing the background color to a lighter shade but going to close this for now given the separation between the h3 element and the background.
